entoas/internal/gen_pets.go

```diff
@@ -0,0 +1,45 @@
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"log"
+
+	"entgo.io/contrib/entoas"
+	"entgo.io/ent/entc"
+	"entgo.io/ent/entc/gen"
+)
+
+func main() {
+	ex, err := entoas.NewExtension(
+		entoas.SpecTitle("My Pets API"),
+		entoas.SpecDescription("Awesome, Mega Cool API to manage Ariel's Pet Leopards!"),
+		entoas.SpecVersion("0.0.1"),
+		// entoas.SpecSecuritySchemes(map[string]spec.SecurityScheme{
+		// 	"apiKeySample": {
+		// 		Type: "apiKey",
+		// 		In:   "header",
+		// 		Name: "X-API-KEY",
+		// 	},
+		// }),
+		// entoas.SpecSecurity([]map[string][]string{
+		// 	{"apiKeySample": {}},
+		// }),
+		// entoas.DumpTo(os.Stdout),
+	)
+	if err != nil {
+		log.Fatalf("creating entoas extension: %v", err)
+	}
+	err = entc.Generate(
+		"./schema",
+		&gen.Config{
+			Target:  "./pets",
+			Package: "entgo.io/contrib/entoas/internal/pets",
```

Comment:
Is it really needed to be explicitly defined?

Reply:
Yes. Resolution of the target does not work correctly otherwise. I have it on my agenda to check if it should be considered a bug or not.

Reply:
Is it because you use a different `Target`? Are you able to reproduce this?